playbook-website/config/webpacker.yml

```diff
@@ -11,7 +11,7 @@ default: &default
 
   # Additional paths webpack should lookup modules
   # ['app/assets', 'engine/foo/app/assets']
-  additional_paths: []
+  additional_paths: ['app']
```

Comment:
Best to configure looking up test matches from here versus custom environment.js